PURL-TYPES.rst

```diff
@@ -390,9 +415,7 @@ Other candidate types to define:
 - ``chef`` for Chef packages:
 - ``chocolatey`` for Chocolatey packages
 - ``clojars`` for Clojure packages:
-- ``cocoapods`` for Cocoapods iOS packages:
```

Comment:
where did this go?

Reply:
Cocoapods was already defined so this seemed duplicative: https://github.com/package-url/purl-spec/blob/b09bb359d503a0d56aea94e03a39c0f54c63f198/PURL-TYPES.rst?plain=1#L53